probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -44,9 +52,24 @@
 
 MIN_FIREFOX_VERSION = 30
 ERROR_CACHE_FILENAME = 'probe_scraper_errors_cache.json'
+ARTIFICIAL_TAG = 'artificial'
 
 
 def load_tags(channel):
+    """
+    A bit of info about `json-tags`
+
+    They are a set of tags that represent the first and
+    last commit hat a release_channel went out on. For example,
```

Comment:
```suggestion
    last commit that a release_channel went out on. For example,
```

But now I want a commit hat.

Reply:
Still says "hat"